tutorials/packaging/create-your-first-snap/create-your-first-snap.md

```diff
@@ -553,15 +522,14 @@ built-in help system. You can discover all the options by running:
 Now run:
 
 ```bash
-snapcraft clean gnu-bash -s build
+snapcraft clean gnu-bash
```

Comment:
Testing this, I did not get the error described here.  Maybe there is a change which eliminates this error and we can simply remove this section (and perhaps find a new example for configflags).  Removing the section will also address the follow-up usage of `snapcraft prime`